Dockerfile.raspberry-pi

```diff
@@ -42,6 +42,10 @@ RUN systemctl enable /etc/systemd/system/resin.service
 RUN git clone https://github.com/gdraheim/docker-systemctl-replacement
 RUN cp docker-systemctl-replacement/files/docker/systemctl3.py /bin/systemctl
 
+RUN rm -f /sbin/runlevel
```

Comment:
Can you leave this out or does COPY require that target file does not exist?

Reply:
yes it does because it's a link and I don't wanna overwrite what it points to